matvienkome/first/task1.py

```diff
@@ -0,0 +1,4 @@
+"""Порахувати кількість пустих списків"""
+
+list1=[]
+for elem in list1:
```

Comment:
"""
working code
"""
i=0
def cheklist(empty_list,i):
    if len(empty_list)>0:
        a=empty_list[0]
        if isinstance(a,list) and len(a)==0:
            i+=1
            return cheklist(empty_list[1:],i)
        return cheklist(empty_list[1:],i)
    return print(i)
cheklist(listt,i)